### ISSUE TITLE
Thesis: Problem Statement

### ISSUE THREAD

**Issue by balacij** [JacquesCarette/Drasil#2920]
I've started posting my work to the [`jasonWriting`](https://github.com/JacquesCarette/Drasil/tree/jasonWriting). With a lot of TODOs currently present, the final rendered PDF is also [posted](https://github.com/JacquesCarette/Drasil/blob/jasonWriting/People/Jason/thesis/thesis.pdf). I will need to take a closer look at the McMaster guidelines to figure out what changes I need to make to the core LaTeX, but this should be okay for drafting right now.

I'm currently working on the Introduction's problem statement specifically. I'm not quite sure how much background discussion of the underlying Drasil theory I should discuss there, so I am going to try to describe the whole background + problem there, and then move components to the "Context: Knowledge Capture & Encoding" section.

Thank you!

**Comment by JacquesCarette** [comment]
Have you been pushing? I have not seen things on that branch?

Can you make the 'issue' clearer? i.e. what do I need to do to "close" this issue?

**Comment by balacij** [comment]
>  Have you been pushing? I have not seen things on that branch?

Yes, the files should be there, perhaps more visible via the [branch compare](https://github.com/JacquesCarette/Drasil/compare/jasonWriting).

>  Can you make the 'issue' clearer? i.e. what do I need to do to "close" this issue?

It's moreso to let you know of this branch, with what's on it right now, and that I'll be posting there. I think we can close this issue any time.

**Comment by JacquesCarette** [comment]
Thanks, the branch compare was very useful. Ok, feedback on what I see there now:
- don't use internal terminology (Chunk, ChunkDB) for subsection names (this is from background) - even as placeholders. Force yourself to find a topic that someone outside Drasil would understand.
- the chapters do make sense
- the stuff on typing seems very much work in progress?

**Comment by balacij** [comment]
Closing this in favour of #2991. I think we've also discussed this elsewhere (via email?), but, to make sure, I changed the section names to avoid internal terminology, and the typing-related chapter is still largely a work in progress.
